operations/bills.md

```diff
@@ -326,9 +327,10 @@ Creates new empty bill assigned to specified account.
 
 #### Bill parameters
 
-| Property | Type | Contract | Description |
-| :-- | :-- | :-- | :-- |
+| Property    | Type | Contract | Description                                                                                                                                                                             |
+|:------------| :-- |:---------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
this is weird, I'd keep the format

Reply:
Yeah, you are right. Rider changing things under my hands. Didn't notice that :)